librz/core/cmd/cmd_debug.c

```diff
@@ -2235,51 +2199,214 @@ RZ_IPI RzCmdStatus rz_cmd_debug_step_until_optype_handler(RzCore *core, int argc
 	return RZ_CMD_STATUS_OK;
 }
 
+// dsue
 RZ_IPI RzCmdStatus rz_cmd_debug_step_until_esil_handler(RzCore *core, int argc, const char **argv) {
 	step_until_esil(core, argv[1]);
 	rz_core_dbg_follow_seek_register(core);
 	return RZ_CMD_STATUS_OK;
 }
 
+// dsuf
 RZ_IPI RzCmdStatus rz_cmd_debug_step_until_flag_handler(RzCore *core, int argc, const char **argv) {
 	step_until_flag(core, argv[1]);
 	rz_core_dbg_follow_seek_register(core);
 	return RZ_CMD_STATUS_OK;
 }
 
+// dt
+RZ_IPI RzCmdStatus rz_cmd_debug_trace_handler(RzCore *core, int argc, const char **argv) {
+	RzDebugTracepoint *t;
+	if ((t = rz_debug_trace_get(core->dbg,
+		     rz_num_math(core->num, argv[1])))) {
+		rz_cons_printf("offset = 0x%" PFMT64x "\n", t->addr);
+		rz_cons_printf("opsize = %d\n", t->size);
+		rz_cons_printf("times = %d\n", t->times);
+		rz_cons_printf("count = %d\n", t->count);
+		// TODO cons_printf("time = %d\n", t->tm);
+	}
+	return RZ_CMD_STATUS_OK;
+}
+
+// dtl
+RZ_IPI RzCmdStatus rz_cmd_debug_traces_handler(RzCore *core, int argc, const char **argv, RzOutputMode mode) {
+	rz_debug_trace_list(core->dbg, mode, core->offset);
+	return RZ_CMD_STATUS_OK;
+}
+
+// dt+
+RZ_IPI RzCmdStatus rz_cmd_debug_trace_add_handler(RzCore *core, int argc, const char **argv) {
+	ut64 addr = rz_num_math(core->num, argv[1]);
+	int count = argc > 2 ? rz_num_math(core->num, argv[2]) : 1;
+	RzAnalysisOp *op = rz_core_op_analysis(core, addr, RZ_ANALYSIS_OP_MASK_HINT);
+	if (op) {
```

Comment:
do 
```
if (!op) {
   RZ_LOG_ERROR("Cannot analyze opcode at 0x%08" PFMT64x "\n", addr);
   return RZ_CMD_STATUS_ERROR;
}
....

instead of if/else.
```